README.rst

```diff
@@ -93,7 +93,7 @@ Basic usage looks like:
 
     # Create a client instance. The timeout and authentication options are not required.
     # Solr URL format: http://host:port/solr/<CORE_NAME>
-    solr = pysolr.Solr("http://localhost:8983/solr/my_core", always_commit=True, [timeout=10], [auth=<type of authentication>])
+    solr = pysolr.Solr("http://localhost:8983/solr/my_core", always_commit=True, timeout=10, auth=<type of authentication>)
```

Comment:
Are the `timeout` and `auth` parameters now mandatory?  Line 94 seems to say that they are optional, so why propose this change?